routers/repo/webhook.go

```diff
@@ -393,6 +402,7 @@ func TestWebhook(ctx *context.Context) {
 	}
 }
 
+// DeleteWebhook delete a webhook
```

Comment:
nitpicking: I think the phrases should be english phrases, like "DeleteWebhook deletes a web hook" (note the ending `s` for third person subject being the function)